### PR TITLE
v0.24.1

### PR DESCRIPTION
**Pull request by jeff-hernandez**
**v0.24.1 May 26, 2021**
* Fixes
  * Update minimum pyyaml requirement to 5.4 (#1433)
  * Update minimum psutil requirement to 5.6.6 (#1438)
* Documentation Changes
  * Update nbsphinx version to fix docs build issue (#1436)
* Testing Changes
  * Create separate workflows for each CI job (#1422)
  * Add minimum dependency checker to generate minimum requirement files (#1428)
  * Add unit tests against minimum dependencies for python 3.7 on PRs and main (#1432, #1445)

### CODE REVIEW

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -2,8 +2,19 @@
...
+    * Fixes
+    * Changes
+    * Documentation Changes
+    * Testing Changes
+
+.. Thanks to the following people for contributing to this release:
+
+v0.24.1 May 26, 2021
+====================
     * Enhancements
```
Comment (gsheni) [inline docs/source/release_notes.rst:18]: Unused header needs to be removed

Comment (gsheni) [reply docs/source/release_notes.rst:18]: Same comment for Changes header

Comment (gsheni) [reply docs/source/release_notes.rst:18]: Also, you can add yourself to the list of contributors

**Review by gsheni — CHANGES_REQUESTED** [review]

File: docs/source/release_notes.rst (hunk 1 above)
Comment (jeff-hernandez) [reply docs/source/release_notes.rst]: Updated

**Review by gsheni — APPROVED** [review]
